securesystemslib/signer/_signature.py

```diff
@@ -0,0 +1,131 @@
+"""Signature container class"""
+
+import logging
+from typing import Any, Dict, Mapping, Optional
+
+logger = logging.getLogger(__name__)
+
+
+class Signature:
+    """A container class containing information about a signature.
+
+    Contains a signature and the keyid uniquely identifying the key used
+    to generate the signature.
+
+    Provides utility methods to easily create an object from a dictionary
+    and return the dictionary representation of the object.
+
+    Attributes:
+        keyid: HEX string used as a unique identifier of the key.
+        signature: HEX string representing the signature.
+        unrecognized_fields: Dictionary of all attributes that are not managed
+            by securesystemslib.
+
+    """
+
+    def __init__(
+        self,
+        keyid: str,
+        sig: str,
+        unrecognized_fields: Optional[Mapping[str, Any]] = None,
+    ):
+        self.keyid = keyid
+        self.signature = sig
+        self.unrecognized_fields: Mapping[str, Any] = unrecognized_fields or {}
```

Comment:
See https://github.com/theupdateframework/python-tuf/issues/1937

Reply:
falsyness point is 100% true.

mapping was my annotation mistake at some point I think: We have already fixed that from similar fields in python-tuf: I'll do  the same thing here.